Terminal.Gui/Views/ComboBox.cs

```diff
@@ -243,7 +243,7 @@ public virtual bool Expand ()
     public event EventHandler Expanded;
 
     /// <inheritdoc/>
-    protected internal override bool OnMouseEvent  (MouseEvent me)
+    protected internal override bool OnMouseEvent (MouseEvent me)
```

Comment:
I still really dislike the name of that class (`MouseEvent`). It's not an event. It's event arguments/state. :man_shrugging: 

Irrelevant to this PR of course.